.gitignore

```diff
@@ -14,3 +14,5 @@ target/
 
 # Ignore state machine binary
 ic-test-state-machine
+
+.vscode
```

Comment:
IDE-specific restrictions?   :slightly_smiling_face:

Reply:
just sneaking in some convenience for myself 🙂